### ISSUE TITLE
Improve reply contexts

### ISSUE THREAD

**Issue by aaronpk** [aaronpk/Aperture#31]
Improve reply contexts by inspecting the reply URL, as documented on https://indieweb.org/reply-context#Minimal_text_reply_contexts

**Comment by kisik21** [comment]
Also maybe give reply context data to Microsub readers, so they would show richer reply contexts as requested in https://github.com/cleverdevil/together/issues/88#issuecomment-446329454

**Comment by kisik21** [comment]
Update on reply contexts: h-cite elements in my posts make reply contexts look richer in Together but not in Monocle. Do I need to file an issue against Monocle for not rendering "h-cite u-in-reply-to" markup?

**Comment by EdwardHinkle** [comment]
Yeah, if Together is showing information that Monocle does not, then Monocle DOES have access to the information and could display it.
